### ISSUE TITLE
 Which data is good to train?

### ISSUE THREAD

**Issue by kimlia545** [clovaai/deep-text-recognition-benchmark#258]
Paper said "This result showed that the diversity of training data can be more important than the number of training examples, and that the effects of using different training datasets is more complex than simply concluding more is better." 
You uesd the MJSynth and SynthText in combination.
I want to train Korean language data. Should I use data with various colors, fonts, backgrounds, widths, gradients, distortions, and blurs?

**Comment by yakhyo** [comment]
I think using rgb images does not help because of the network input has one channel default . However you can change it by `opt.rgb=True`.

**Comment by kimlia545** [comment]
@yakhyo 
Thanks

**Comment by bit-scientist** [comment]
Hi, @kimlia545, do you happen to have a pretrained model for Korean (or Korean + English) language that you can share? As their site only supports around 10 tests per day, I would like to have a separate model on premise. Thank you!
